app/views/admin/cms/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :cms_heading do %>
```

Comment:
interesting, why is all the content being rendered into `content_for` rather than just output normally?